Libplanet.Explorer.Executable/Program.cs

```diff
@@ -98,21 +93,15 @@ public static async Task Main(string[] args)
                     var privateKey = new PrivateKey();
 
                     // FIXME: The appProtocolVersion should be fixed properly.
-                    var swarmOptions = new SwarmOptions
-                    {
-                        MaxTimeout = TimeSpan.FromSeconds(10),
-                    };
-
                     swarm = new Swarm<AppAgnosticAction>(
                         blockChain,
                         privateKey,
                         options.AppProtocolVersionToken is string t
                             ? AppProtocolVersion.FromToken(t)
                             : default(AppProtocolVersion),
-                        workers: 50,
                         differentAppProtocolVersionEncountered: (p, pv, lv) => true,
                         iceServers: new[] { options.IceServer },
-                        options: swarmOptions
+                        workers: options.Workers
```

Comment:
Is it okay we stash `options` param? cc @earlbread

Reply:
It is my mistake. I will recover it now. 😿